vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -153,6 +158,18 @@ function Get-FilesToCacheOnVHD {
 
         foreach ($URL in $map[$dir]) {
             $fileName = [IO.Path]::GetFileName($URL)
+
+            if ($dir.StartsWith())
+            # Windows containerD supports Windows containerD, starting from Kubernetes 1.20
+            if ($containerRuntime -eq 'containerd' -And $dir -eq "c:\akse-cache\win-k8s\") {
```

Comment:
indent

Reply:
Changes should happen in test.ps1 but forgot to remove it from here.

Reply:
Do you mean I should start `{` in a new line? I found the code around also  using '{' in the same line as the logic check, while in test ps, we have a different pattern.